src/client/pages/Forum.js

```diff
@@ -80,6 +19,41 @@ function Forum() {
     document.body.style.overflowY = 'unset';
   }
 
+  function queryPosts() {
+    fetch('http://localhost:5000/api/forum/posts')
+      .then(res => res.json())
+      .then(
+        (serverResult) => {
+          JSON.stringify();
```

Comment:
The `JSON.stringify();` does not do anything in this case. We can simply remove it.

Reply:
So I copy pasted basically this whole function from the SearchResults component which essentially does the same thing. Should the JSON.stringify() not be there either?

Reply:
removed it either way

Reply:
That ` JSON.stringify()` has been taken care of.